internal/nodes/avi_model_evh_nodes.go

```diff
@@ -1083,7 +1097,18 @@ func (o *AviObjectGraph) BuildModelGraphForSecureEVH(routeIgrObj RouteIngressMod
 	}
 	if !certsBuilt {
 		certsBuilt = o.BuildTlsCertNodeForEvh(routeIgrObj.GetSvcLister(), vsNode[0], namespace, tlssetting, key, infraSettingName, host)
+	} else {
+		//Delete sslcertref object if host crd sslcertref (sslcertAviRef) is present for given host
+		secretName := tlssetting.SecretName
+		if strings.HasPrefix(secretName, lib.RouteSecretsPrefix) {
+			// Openshift case
+			vsNode[0].DeleteCACertRefInEVHNode(lib.GetCACertNodeName(infraSettingName, host), key)
```

Comment:
In case of openshift route, cacert would be referred by a ssl key cert. I think we would have to remove both.

Reply:
Yes. Thanks. Added deletion of CA for openshift and deletion of sslkeycert for both openshift and k8.